Native-Frameworks/Screens/DetailView/FrameworkDetailView.swift

```diff
@@ -27,7 +27,7 @@ struct FrameworkDetailView: View {
             } label: {
                 FrameworksButton(title: "More Info")
             }
-        }.fullScreenCover(isPresented: $isShowingSafariView, content: {
+        }.sheet(isPresented: $isShowingSafariView, content: { // TODO: You show screen `fullScreenCover` under `fullScreenCover`, I think you should call only 1 time, probably in first place in your Grid you should use list and push your screen 
```

Comment:
@jantimar I changed to `sheet` to not call `fullScreenCover` twice. I want to later implement changing from `GridView` to `ListView` with my `popup`.